tests/system/Autoloader/AutoloaderTest.php

```diff
@@ -284,29 +294,45 @@ public function testFindsComposerRoutesWithComposerPathNotFound()
         $modules                     = new Modules();
         $modules->discoverInComposer = true;
 
-        $this->loader = new Autoloader();
+        $loader = new Autoloader();
 
         rename(COMPOSER_PATH, COMPOSER_PATH . '.backup');
-        $this->loader->initialize($config, $modules);
+        $loader->initialize($config, $modules);
         rename(COMPOSER_PATH . '.backup', $composerPath);
 
-        $namespaces = $this->loader->getNamespace();
+        $namespaces = $loader->getNamespace();
         $this->assertArrayNotHasKey('Laminas\\Escaper', $namespaces);
     }
 
     public function testAutoloaderLoadsNonClassFiles(): void
     {
-        $config = new Autoload();
-
+        $config          = new Autoload();
         $config->files[] = SUPPORTPATH . 'Autoloader/functions.php';
 
-        $this->loader = new Autoloader();
-        $this->loader->initialize($config, new Modules());
-        $this->loader->register();
+        $loader = new Autoloader();
+        $loader->initialize($config, new Modules());
+        $loader->register();
 
         $this->assertTrue(function_exists('autoload_foo'));
         $this->assertSame('I am autoloaded by Autoloader through $files!', autoload_foo());
         $this->assertTrue(defined('AUTOLOAD_CONSTANT'));
         $this->assertSame('foo', AUTOLOAD_CONSTANT);
+
+        $loader->unregister();
+    }
+
+    public function testLoadHelpers()
+    {
+        $config            = new Autoload();
+        $config->helpers[] = 'form';
+
+        $loader = new Autoloader();
+        $loader->initialize($config, new Modules());
+
```

Comment:
Does this need to be `@runInSeparateProcess` to be definitive? One way to check:
```suggestion
        $this->assertFalse(function_exists('form_open'));

```

Reply:
Yes! Done.